internal/core/src/segcore/FieldIndexing.cpp

```diff
@@ -142,6 +149,8 @@ CreateIndex(const FieldMeta& field_meta, const SegcoreConfig& segcore_config) {
             return std::make_unique<ScalarFieldIndexing<float>>(field_meta, segcore_config);
         case DataType::DOUBLE:
             return std::make_unique<ScalarFieldIndexing<double>>(field_meta, segcore_config);
+        case DataType::VarChar:
```

Comment:
why not use "VARCHAR" just like other types ?

Reply:
fix